mimic/model/identity.py

```diff
@@ -20,3 +26,205 @@ def from_store(cls, session_store, tenant_id):
 
     identity_admin = attr(default=False,
                           validator=validators.instance_of(bool))
+
+
+class ICredentials(Interface):
```

Comment:
This is really just me thinking out loud, but - wouldn't this interface be singular? `ICredential`?  Each token or password or whatever is just one credential…